src/simplejavatexteditor/SupportedKeywords.java

```diff
@@ -12,7 +12,7 @@
  */
 public class SupportedKeywords {
 
-    private String[] supportedLangage = {".cpp",".java"};
+    private String[] supportedLanguage = {".cpp",".java"};
```

Comment:
Thanks for fixing the typo! You could also change it to plural to be `supportedLanguages` to be clearer.

Reply:
@LordStephen77 Did you get a chance to look at my comment?